tools/ci.psm1

```diff
@@ -156,25 +156,24 @@ Function Set-BuildVariable
 }
 
 # Emulates running all of CI but locally
-function Invoke-AppVeyorFull
+function Invoke-CIFull
```

Comment:
Maybe this should be called Invoke-WindowsInstall, since AppVeyor was Windows only and CI is too generic at this moment, as Linux and macOS still have their own cmdlets in CI.psm1.
Same for all other renaming.

Reply:
Install is wrong as install in this means BootStrap

Full means the full CI run.  I think this is the correct name.  We should assume that Linux and macOS get merged in the future.